modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -61,9 +61,10 @@ def test_from_sql_distributed(make_sql_connection):  # noqa: F811
                 upper_bound=6,
                 max_sessions=2,
             )
-
-        df_equals(modin_df_from_query, pandas_df)
-        df_equals(modin_df_from_table, pandas_df)
+            # If read operations are asynchronous, then the dataframes
+            # check should be inside `ensure_clean_dir` context
```

Comment:
```suggestion
            # check should be inside `ensure_clean_dir` context
            # because the file may be deleted before actual reading starts
```